fedimint-api/src/tiered_multi.rs

```diff
@@ -103,6 +104,41 @@ impl<T> TieredMulti<T> {
     pub fn get_mut(&mut self, amt: Amount) -> Option<&mut Vec<T>> {
         self.0.get_mut(&amt)
     }
+
+    /// Determines the denominations to use when representing an amount
```

Comment:
I'm not sure we should make it a method on `TieredMulti`. We use `TieredMulti` for bunch of things, and it's not very intuitive to find it here. The fact that `&self` means "denominations user already have" is also not going to be very intuitive for people scanning through this method being used.

Possibly this should be just a standalone function in some utility module somewhere (with the mint client side module maybe).

Tieres would probably be best taken as `IntoIterator<Item = Amount>`. Number of existing coins as `Tiered<usize>` (with `TieredMulti::tier_counts(&self) -> Tiered<usize>` for an easy conversion maybe?

Reply:
Good point, I was not sure where to put it, but I can refactor.